pkg/nmstatectl/remote.go

```diff
@@ -0,0 +1,68 @@
+package nmstatectl
```

Comment:
This should not be a part of this PR, right?

Reply:
Removed, this is from the PoC.